src/main/java/bio/terra/service/filedata/azure/AzureSynapsePdao.java

```diff
@@ -1183,15 +1183,14 @@ private Object extractValue(ResultSet resultSet, Column column) {
         return switch (column.getType()) {
           case BOOLEAN -> resultSet.getBoolean(column.getName());
           case BYTES -> resultSet.getBytes(column.getName());
-          case DATE -> resultSet.getDate(column.getName());
-          case DIRREF, FILEREF, STRING, TEXT -> resultSet.getString(column.getName());
+          case DIRREF, FILEREF, STRING, TEXT, DATE, DATETIME, TIMESTAMP -> resultSet.getString(
+              column.getName());
```

Comment:
Is the testing covered in `AzureSynapsePdaoConnectedTest`?  Might need to be updated to account for this change